core/src/main/java/hudson/Functions.java

```diff
@@ -1884,6 +1885,30 @@ public String getPasswordValue(Object o) {
         return o.toString();
     }
 
+    /**
+     * Retrieve default value to be used when setting field on type.
+     * Used by {@code <f:textbox>} to retrieve default value defined by component meta-data.
+     */
+    public String getDefaultValue(Class type, String field) {
```

Comment:
Could you implement some unit tests for this method, especially cases for the catch statement due to the complexity ?